test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_ListDescribeConsumerGroups.cs

```diff
@@ -63,16 +63,68 @@ private void checkConsumerGroupDescription(
         [Theory, MemberData(nameof(KafkaParameters))]
         public void AdminClient_ListDescribeConsumerGroups(string bootstrapServers)
         {
+            var groupID = Guid.NewGuid().ToString();
```

Comment:
Please add another similar test creating `Classic` group

Reply:
Actually the way 848 integration testing work is that we run 2 instances of Semaphore CI where one version runs for all the test having no broker min version, so assume only classic group protocol is used. 
Now currently semaphore will only run some parts of test that needs verifying which means with broker version > 3.8.0.0 and environment variable for the consumer group protocol chosen to be consumer type.